### ISSUE TITLE
add support for preregistered studies

### ISSUE THREAD

**Issue by monperrus** [emsejournal/openscience#5]
We can support registered report / preregistered studies. Basically, reviewers review a study plan, and if approved, regardless of whether the results are positive or negative, the study will be published. There’s a proposal to do so at CSCW: https://blog.communitydata.cc/a-proposal-to-mitigate-false-discovery-in-cscw-research/ (suggestion by Neil Ernst)

See also https://en.wikipedia.org/wiki/Registered_report

**Comment by robertfeldt** [comment]
Neil is running an RR track for MSR'20. I suggest we try to make that a very successful one, learn from that, and then discuss what (if any) are the next steps forward.

@neilernst do you have some page or CfP for the track yet? Please link to it here or on our links page when ready.

**Comment by neilernst** [comment]
not yet, but maybe it makes sense to host the page on this repo? As it is an EMSE initiative as well. The challenge with the conference model is each year MSR would be managed by a new team.

**Comment by robertfeldt** [comment]
Yes, would be great to host it here. We can also link to it from a web page we'll do on our emsejournal.github.io page that lists all our Special Issue's. Future teams can be handled via PR or we just link out to whatever site they decide to use. As long as EMSE is the journal receiving the finalized RR papers having it here makes sense.

**Comment by neilernst** [comment]
see https://github.com/emsejournal/openscience/blob/master/registered-reports.md
